central/buildbot.py

```diff
@@ -22,18 +22,18 @@ def make_netstring(s):
     return str(len(s)).encode('ascii') + b':' + s + b','
 
 
-def make_build_request(repo, pr_id, job_id, baserev, headrev, patch, who,
+def make_build_request(repo, pr_id, job_id, baserev, headrev, who,
                        comment):
     """Creates a build request binary blob in the format expected by the
     buildbot."""
 
     request_dict = {
-        'branch': '',
+        'branch': 'refs/pull/%d/merge' % pr_id,
```

Comment:
Okay, I changed this to `refs/pull/%d/head` and it works fine, I'll leave the building-merge-refs idea until some time in the future.